cli/src/test/snapshots/unlock.test.ts.md

```diff
@@ -0,0 +1,63 @@
+# Snapshot report for `src/test/unlock.test.ts`
+
+The actual snapshot is saved in `unlock.test.ts.snap`.
+
+Generated by [AVA](https://ava.li).
+
+## lock command fails when app name not in local project dir
+
+> Snapshot 1
+
+    {
+      err: ` ›   Error: No app in current working directory, ␊
+       ›   /private/var/folders/31/dd8tb9gn35x27gpv7c1200yh0000gn/T/dot-reshuffle-sYF␊
```

Comment:
This snapshot will *always* break on another machine.  You'll have to replace it, e.g. to delete the pathname from `err`.
Also I don't understand how we got the next line, is that some oclif weirdness?